fearless/Modules/AllDone/AllDoneViewController.swift

```diff
@@ -77,6 +77,7 @@ extension AllDoneViewController: AllDoneViewInput {
 
     func didReceive(explorer: ChainModel.ExternalApiExplorer?) {
         rootView.subscanButton.isHidden = explorer == nil
```

Comment:
I suggest group these buttons inside custom component with "func updateVisibility(for explorer: ChainModel.ExternalApiExplorer?)". Reuse this component for SwapTransactionDetail module